content/en/docs/reference/using-api/cel.md

```diff
@@ -61,13 +61,38 @@ Example CEL expressions:
-
-- CEL standard functions, defined in the [list of standard definitions](https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions)
-- CEL standard [macros](https://github.com/google/cel-spec/blob/v0.7.0/doc/langdef.md#macros)
-- CEL [extended string function library](https://pkg.go.dev/github.com/google/cel-go/ext#Strings)
+## CEL options, language features, and libraries
+
+CEL is configured with the following options, libraries and language features, introduced at the specified Kubernetes versions:
+
+| CEL option, library or language feature             | Included                                                                                                        | Availablity                |
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | -------------------------- |
+| [Standard macros](stdmacros)                        | `has`, `all`, `exists`, `exists_one`, `map`, `filter`                                                           | All Kubernetes versions    |
+| [Standard functions](stdlib)                        | See [official list of standard definitions](stdlib)                                                             | All Kubernetes versions    |
+| [Homogeneous Aggregate Literals](opt1)              |                                                                                                                 | All Kubernetes versions    |
+| [Default UTC Time Zone](opt2)                       |                                                                                                                 | All Kubernetes versions    |
+| [Eagerly Validate Declarations](opt3)               |                                                                                                                 | All Kubernetes versions    |
+| [extended strings library](stringsgodoc), Version 1 | `charAt`, `indexOf`, `lastIndexOf`, `lowerAscii`, `upperAscii`, `replace`, `split`, `join`, `substring`, `trim` | All Kubernetes versions    |
+| Kubernetes list library                             | See [Kubernetes list library](#kubernetes-list-library)                                                         | All Kubernetes versions    |
+| Kubernetes regex library                            | See [Kubernetes regex library](#kubernetes-regex-library)                                                       | All Kubernetes versions    |
+| Kubernetes URL library                              | See [Kubernetes URL library](#kubernetes-url-library)                                                           | All Kubernetes versions    |
+| Kubernetes authorizer library                       | See [Kubernetes authorizer library](#kubernetes-authorizer-library)                                             | All Kubernetes versions    |
```

Comment:
This is not All Kubernetes versions if I remember correctly. It should be introduced in 1.27?

Reply:
The only place where we allow the authorizer variable can be used is ValidatingAdmissionPolicy, and that API was introduced in 1.27.

So while it's true that the library can't be used before 1.27, there is no need to document it as a version bound library, which is why I didn't bother to put a version bound here.   Basically what this is saying is "If you have access to authorizer, you have access to the library".